lib/src/rows/Rows.stories.tsx

```diff
@@ -0,0 +1,314 @@
+        <DxcRows.Row>
+          <Placeholder width="medium" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="large" />
+        </DxcRows.Row>
+      </DxcRows>
+    </Container>
+    <Title title="AlignX = baseline" theme="light" level={4} />
+    <Container>
+      <DxcRows alignX="baseline">
+        <DxcRows.Row>
+          <Placeholder width="small" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="medium" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="large" />
+        </DxcRows.Row>
+      </DxcRows>
+    </Container>
+    <Title title="AlignX = stretch" theme="light" level={4} />
+    <Container>
+      <DxcRows alignX="stretch">
+        <DxcRows.Row>
+          <Placeholder width="auto" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="medium" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="large" />
+        </DxcRows.Row>
+      </DxcRows>
+    </Container>
+    <Title title="AlignY = end" theme="light" level={4} />
+    <Container>
+      <DxcRows alignY="end">
+        <DxcRows.Row>
+          <Placeholder width="small" height="small" />
+          <Placeholder width="medium" height="medium" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="medium" height="medium" />
+          <Placeholder width="small" height="small" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="large" height="large" />
+          <Placeholder width="small" height="small" />
+        </DxcRows.Row>
+      </DxcRows>
+    </Container>
+    <Title title="AlignY = center" theme="light" level={4} />
+    <Container>
+      <DxcRows alignY="center">
+        <DxcRows.Row>
+          <Placeholder width="small" height="small" />
+          <Placeholder width="medium" height="medium" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="medium" height="medium" />
+          <Placeholder width="small" height="small" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="large" height="large" />
+          <Placeholder width="small" height="small" />
+        </DxcRows.Row>
+      </DxcRows>
+    </Container>
+    <Title title="AlignY = baseline" theme="light" level={4} />
+    <Container>
+      <DxcRows alignY="baseline">
+        <DxcRows.Row>
+          <Placeholder width="small" height="small" />
+          <Placeholder width="medium" height="medium" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="medium" height="medium" />
+          <Placeholder width="small" height="small" />
+        </DxcRows.Row>
+        <DxcRows.Row>
+          <Placeholder width="large" height="large" />
+          <Placeholder width="small" height="small" />
+        </DxcRows.Row>
+      </DxcRows>
+    </Container>
+    <Title title="AlignY = stretch" theme="light" level={4} />
```

Comment:
The Placeholder is overflowing with `alignY="strech"`.

![image](https://user-images.githubusercontent.com/44321109/175334554-dfed1844-daec-416c-b518-553c6a93f6e8.png)

Reply:
Done!